<?xml version="1.0"?>
<launch>
	<include file="$(find brl_teleop_p3dx)/launch/p3dx.launch" />
	<node name="MMSS_slave" pkg="brl_teleop_controller" type="mobile_robot_multi_controller" output="screen">
		<param name="slave" value="true"	/>
		<param name="popc_enable" value="fasle"	/>
		<param name="time_delay" value="1"	/>

		<param name="position_master1_slave_src_name" value="/MMSS_master1" />
    	<param name="position_master1_slave_topic_name" value="position_master_slave" />

		<param name="position_master2_slave_src_name" value="/MMSS_master2" />
    	<param name="position_master2_slave_topic_name" value="position_master_slave" />
		
		<param name="velocity_master1_slave_src_name" value="/MMSS_master1" />
    	<param name="velocity_master1_slave_topic_name" value="velocity_master_slave" />

		<param name="velocity_master2_slave_src_name" value="/MMSS_master2" />
    	<param name="velocity_master2_slave_topic_name" value="velocity_master_slave" />

		<param name="actual_velocity_src_name" value="/p3dx" />
    	<param name="actual_velocity_topic_name" value="velocity" />
    	
		<rosparam param="gains">{'P': 0.2, 'D': 0.5}</rosparam>

	</node>
</launch>
